packages/@atjson/document/src/serialize.ts

```diff
@@ -446,7 +448,22 @@ export function serialize(
   lastIndex = 0;
   for (let token of tokens) {
     if (parseTokens.length === 0) {
-      text += doc.content.slice(lastIndex, token.index);
+      let chunk = doc.content.slice(lastIndex, token.index);
+      let reservedCharacterIndex = chunk.indexOf("\uFFFC");
+      if (reservedCharacterIndex !== -1) {
+        throw new Error(
+          `Text contains reserved character +uFFFC at index ${
+            lastIndex + reservedCharacterIndex
+          }.\n\n${doc.content.slice(
+            Math.max(0, lastIndex + reservedCharacterIndex - 25),
```

Comment:
Could we add a comment here? I'm guessing there are five object replacement characters here but I'm having trouble following what they are and where they come from

Reply:
Ahh, I see— this is definitely possible. This was more for context, but I should definitely add some sort of _this one, here_ in that comment.

Reply:
Ok, I added more info to this